### ISSUE TITLE
Not worked in android 5

### ISSUE THREAD

**Issue by pere69** [ViktorDegtyarev/CallRecLib#7]
I tested this source
i android 7 worked.... but in android 5 .1 don't save records

**Comment by chwakil** [comment]
Is it working in Android 7.1.1

**Comment by KnightOfRealm** [comment]
yes it is working on Android 7.1.1. But only for the 1st run then it stops as it's NDK part crashes and in android 5.1 it does not for the same reason. It is something to do with the start and stop function.
I am trying to resolve it. All help will be appreciated.
